ukrainian/break-through-wall/try-the-taste/continue-eating/wait/wait.md

```diff
@@ -1,2 +1,6 @@
 Ви чекаєте. І тут раптом чуєте гуркіт, обертаєтесь і бачите, що стіна пробита.
-Через отвір видно  постать гнома. Він запрошує вас зайти до себе.
\ No newline at end of file
+Через отвір видно  постать гнома. Він запрошує вас зайти до себе.
+
+Приймаєте запрошення гнома і [заходите](go-with-dwarf/go-with-dwarf.md) у діру...
+
+З диким ревом б'єте гнома по голові і знову намагаєтеся [пробити стіну](break-through-wall-again/break-through-wall-again.md)...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.